doc/specs/stdlib_string_type.md

```diff
@@ -1350,7 +1350,7 @@ program demo
   use stdlib_string_type
   implicit none
   type(string_type) :: string, reverse_string
-  
```

Comment:
These `double spaces` are redundant because the two `Enters` did the job here of creating a new line